src/test/java/com/lpvs/entity/LPVSLicenseTest.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2022, Samsung Electronics Co., Ltd. All rights reserved.
+ *
+ * Use of this source code is governed by a MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package com.lpvs.entity;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class LPVSLicenseTest {
+    LPVSLicense lpvsLicense;
+    final Long baseLicenseId = 1234567890L;
+    final String baseLicenseName = "licenseName";
+    final String baseSpdxId = "spdxId";
+    final String baseAccess = "access";
+    final String baseChecklistUrl = "checklistUrl";
+    List<String> baseIncompatibleWith = Arrays.asList("incompatibleWith1", "incompatibleWith2", "incompatibleWith3");
+
+
+    @BeforeEach
+    void setUp() {
+        lpvsLicense = new LPVSLicense(baseLicenseId,
+                baseLicenseName,
+                baseSpdxId,
+                baseAccess,
+                baseChecklistUrl,
+                baseIncompatibleWith);
+    }
+
+    @Test
+    public void constructorTest() {
+        LPVSLicense lpvsLicenseBase = new LPVSLicense(lpvsLicense);
+        assertEquals(lpvsLicenseBase.getLicenseId(), baseLicenseId);
+        assertEquals(lpvsLicenseBase.getLicenseName(), baseLicenseName);
+        assertEquals(lpvsLicenseBase.getSpdxId(), baseSpdxId);
+        assertEquals(lpvsLicenseBase.getAccess(), baseAccess);
+        assertEquals(lpvsLicenseBase.getChecklistUrl(), baseChecklistUrl);
+//        assertIterableEquals(lpvsLicenseBase.getIncompatibleWith(), baseIncompatibleWith);
```

Comment:
I checked we do not use the constructor `new LPVSLicense(lpvsLicense)` in the code. 
So please:
1. Remove this constructor from the file `\src\main\java\com\lpvs\entity\LPVSLicense.java`
2. Replace the constructor in your unit test with this one:
`LPVSLicense(Long licenseId, String licenseName, String spdxId, String access, String checklistUrl, List<String> incompatibleWith)`